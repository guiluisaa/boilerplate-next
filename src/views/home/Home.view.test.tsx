import { render, screen } from '@testing-library/react';

import HomeView from './Home.view';

describe('view | <HomeView />', () => {
  it('should match snapshot', () => {
    const { container } = render(<HomeView />);

    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render the heading', () => {
    render(<HomeView />);

    expect(
      screen.getByRole('heading', { name: /home view/i })
    ).toBeInTheDocument();
  });
});
