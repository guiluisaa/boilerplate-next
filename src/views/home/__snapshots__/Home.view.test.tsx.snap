// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`view | <HomeView /> should match snapshot 1`] = `
<div
  class="Layoutcomponent__Wrapper-sc-hjilh-0 kVDPjA"
>
  <a
    href="/posts"
  >
    Posts
  </a>
  <h1
    class="Homeview__Title-sc-1pmkhfg-0 ewFurc"
  >
    title
  </h1>
  <a
    href="/"
  >
    PortuguÃªs do Brasil
  </a>
  <br />
  <a
    href="/"
  >
    Enlish
  </a>
</div>
`;
